extensions/blocks/podcast-player/edit.js

```diff
@@ -49,14 +62,20 @@ const supportUrl =
 const PodcastPlayerEdit = ( {
 	attributes,
 	setAttributes,
+
```

Comment:
Unnecessary line break

Reply:
I wanted to order attributes by kind. Let me remove these lines :-(